### PR TITLE
Ccarfi tales of ronin blog

### PR DESCRIPTION
**Pull request by ccarfi**
tales of ronin blog post

### CODE REVIEW

**Comment by julien51** [comment]
@clemsos can we make sure we run the deployment job on these PR as well (our setup should deploy static sites as "drafts" which means they don't go through the "main" website(s) but can still be tested!)

**Comment by ccarfi** [comment]
@julien51 @searchableguy @clemsos can one of you please review at your convenience? thanks!

**Review by julien51 — APPROVED** [review]

**Comment by julien51** [comment]
Preview : https://621695dade3e4635d3d76e4b--unlock-protocol-com.netlify.app/blog/talesofelatora

File: unlock-protocol-com/blog/talesofelatora.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+title: A Community Unlocks the Tales of Elatora
+subTitle: One artist’s childhood dream comes to life through NFTs as both memberships and art
+authorName: Crystal Street
+publishDate: February 21, 2022
+description: In the hands of a visionary artist, Unlock Protocol helped @littlefortunes create a stunning work of art that is also living, breathing fantasy story written in collaboration with the Tales of Elatora NFT collectors.
+image: /images/blog/talesofronin/tales-of-ronin-cover-min.jpeg
+---
+
+*As an open-source protocol, we take the tools we’ve built and place them in the hands of creators and developers and hope something beautiful emerges. In the late fall of 2021, Caroline aka [@littlefortunes](https://twitter.com/littlefortunes) joined our Discord with a brilliant idea, a ton of passion, and the determination to find solutions for the project she was building. She took her idea, integrated Unlock, and launched an initial rare NFT collection that sold out in 24 hours. An amazing accomplishment in itself, but what she’s created with her artistic brilliance and perseverance deserves a deeper examination.*
+
```
Comment (julien51) [inline unlock-protocol-com/blog/talesofelatora.md:11]:
```suggestion

![Tales of Elatora](/images/blog/talesofronin/tales-of-ronin-cover-min.jpeg)


```

**Pull request merged**